packages/plugins/Scale/src/components/Scale.vue

```diff
@@ -4,6 +4,23 @@
     :title="$t('common:plugins.scale.label')"
     :aria-label="$t('common:plugins.scale.label')"
   >
+    <select
+      v-if="showSelectOptions"
+      v-model="scale"
+      :title="$t('common:plugins.scale.scaleSwitcher')"
+      :aria-label="$t('common:plugins.scale.scaleSwitcher')"
+      class="scale-as-a-ratio scale-switcher"
+      @change="setZoomLevelByScale($event.target.selectedIndex)"
+    >
+      <option
+        v-for="(option, i) in zoomOptions"
+        :key="i"
+        :value="option"
+        class="scale-as-a-ratio"
+      >
+        {{ i === 0 ? option : scaleNumberToScale(option.scale) }}
+      </option>
+    </select>
     <span class="scale-as-a-ratio">
```

Comment:
The problem here (which was why I implemented a seperate switcher and a scale display instead of one element that does both) was, that the calculated scale would not be in the options selection, so the select element would not display the current scale. I changed it now so that the next best option.scale from the options selection is displayed. But that means that the displayed scale might differ from the calculated scale. I'm not sure about this solution, but maybe this is ok for the moment?

Reply:
commit
'show next best matching scale of zoom options if no mathcing scale is found'

Reply:
I think this seems fine for now. I'll have that in mind when writing the issue regarding the dpi problem of scale. 🎩